### PR TITLE
Create FAQ entry about methods and parameters

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As asked in https://t.me/grammyjs/108084

### CODE REVIEW

**Review by roj1512 — APPROVED** [review]

File: site/docs/resources/faq.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ Here is a collection of frequently asked questions regarding [common errors](#wh
 
 If this FAQ does not answer your question, you should also have a look at the [Bot FAQ](https://core.telegram.org/bots/faq) written by the Telegram team.
 
+## Where can I find docs about a method?
```
Comment (KnorpelSenf) [inline site/docs/resources/faq.md:7]: Shouldn't this be title case?

Comment (roj1512) [reply site/docs/resources/faq.md:7]:
```suggestion
## Where Can I Find Docs About a Method?
```

File: site/docs/resources/faq.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ Here is a collection of frequently asked questions regarding [common errors](#wh
...
+
+In the API reference.
+You probably want to understand [this](../guide) better.
+
+## A method is missing a parameter!
```
Comment (roj1512) [inline site/docs/resources/faq.md:12]:
```suggestion
## A Method Is Missing a Parameter!
```

File: site/docs/resources/faq.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ Here is a collection of frequently asked questions regarding [common errors](#wh
...
+
+No, it's not.
+
+Step 1: Make sure you have the latest grammY version installed.
+
+Step 2: Check if the parameter is optional.
+If it is, then grammY will collect it in the options object called `other`.
+Pass `{ parameter_name: value }` in that place and it'll work.
+As always, TypeScript will auto-complete the parameter names for you.
```
Comment (roj1512) [inline site/docs/resources/faq.md:21]:
```suggestion
1. Make sure you have the latest grammY version installed.

2. Check if the parameter is optional.
     If it is, then grammY will collect it in the options object called `other`.
     Pass `{ parameter_name: value }` in that place and it'll work.
     As always, TypeScript will auto-complete the parameter names for you.
```

**Review by roj1512 — APPROVED** [review]

**Review by CikiMomogi — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Pull request merged**